module/Finna/src/Finna/RecordDriver/SolrEad3.php

```diff
@@ -47,6 +47,12 @@
  */
 class SolrEad3 extends SolrEad
 {
+    const TYPE_IMAGE_FULLRES = 'Bittikartta - Fullres - Jakelukappale';
```

Comment:
Näiden pitäisi olla konffattavia tietolähdekohtaisesti tai ainakin array-muodossa niin, että voidaan helposti lisätä vaihtoehtoja. Muuten jos tulee useammasta lähteestä, niin on aika iso mahdollisuus, etteivät toimi yhteen. 

Tässä tiedostossa on hirveä määrä stringejä muutenkin. Ne voisi nyt tässä vaiheessa ainakin kerätä const-taulukoiksi, joihin saa tarvittaessa lisättyä.

Reply:
Jep, sekava on. Siirsin suurimman osan stringeistä näin aluksi consteiksi.

Reply:
Ok.